next.clayui.com/src/templates/components.js

```diff
@@ -0,0 +1,341 @@
+/**
```

Comment:
Later we can remove this template and use `docs`. It is not necessary to have this.